### ISSUE TITLE
a question about the multi-task loss function

### ISSUE THREAD

**Issue by machine981** [maudzung/TTNet-Realtime-for-Table-Tennis-Pytorch#24]
In the code of the multi-task loss function, I would like to know why the classification loss is calculated in that way.

           ` total_loss += local_ball_loss / (torch.exp(2 * self.log_vars[log_vars_idx])) + self.log_vars[log_vars_idx]`

In that multi_task_loss paper (2018CVPR), author calculated classification loss through cross_entropy(CE) scaled by sigma^2 then plus log(sigma). But in your code, it seems to be calculated through CE scaled by sigma^4. I wanna know whether It's a mistake or a trick. Thanks.
